src/components/Table/Table.vue

```diff
@@ -69,4 +84,9 @@ const headers = computed<Field[]>(() => {
             .map(f => ({key: f}))
         : [];
 });
+
+const {classes} = useClasses(computed(() => [
+    props.hover && 'table-hover',
+    props.click && 'table-click',
```

Comment:
Interesting

Reply:
it's one line if condition without the else!
in PHP we have 
```php
$props->hover ?: 'table-hover'
```

but in any other languages `props.hover && 'table-hover'` will work